libbeat/cmd/instance/beat_test.go

```diff
@@ -97,12 +97,7 @@ func TestInitKibanaConfig(t *testing.T) {
 	username, err := kibanaConfig.String("username", -1)
 	assert.NoError(t, err)
 	password, err := kibanaConfig.String("password", -1)
-<<<<<<< HEAD
-=======
 	assert.NoError(t, err)
-	api_key, err := kibanaConfig.String("api_key", -1)
```

Comment:
Did you mean to remove the actual tests as well?

Reply:
Yes, this test was added recently and does not exist in 7.15. This was the cause of the merge conflict as can be seen in the files changed overview.